lms/templates/instructor/instructor_dashboard_2/edit_coupon_modal.html

```diff
@@ -3,7 +3,8 @@
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
 %>
-<section id="edit-coupon-modal" class="modal" role="dialog" tabindex="-1" aria-label="${_('Edit Coupon Code')}">
+<section id="edit-coupon-modal" class="modal" role="dialog" tabindex="-1" aria-labelledby="header-edit-coupon-code">
+    <h2 id="header-edit-coupon-code">${_("Edit Coupon Code")}</h2>
```

Comment:
i also see repetition of the id with the `aria-labelledby`. can this be more programmatic?